packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -1398,22 +1396,27 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
         );
       });
 
-      it('should execute "eth_sendRawTransaction" and deploy a contract with more than max transaction fee', async function () {
-        const gasPrice = await relay.gasPrice(requestId);
-        const transaction = {
-          type: 2,
-          chainId: Number(CHAIN_ID),
-          nonce: await relay.getAccountNonce(accounts[2].address, requestId),
-          maxPriorityFeePerGas: gasPrice,
-          maxFeePerGas: gasPrice,
-          gasLimit: Constants.MAX_GAS_PER_SEC,
-          data: '0x' + '00'.repeat(60000),
-        };
-        const signedTx = await accounts[2].wallet.signTransaction(transaction);
-        const error = predefined.INTERNAL_ERROR();
+      if (!useAsyncTxProcessing) {
+        it('should execute "eth_sendRawTransaction" and deploy a contract with more than max transaction fee', async function () {
+          const gasPrice = await relay.gasPrice(requestId);
+          const transaction = {
+            type: 2,
+            chainId: Number(CHAIN_ID),
+            nonce: await relay.getAccountNonce(accounts[2].address, requestId),
+            maxPriorityFeePerGas: gasPrice,
+            maxFeePerGas: gasPrice,
+            gasLimit: Constants.MAX_GAS_PER_SEC,
+            data: '0x' + '00'.repeat(60000),
+          };
+          const signedTx = await accounts[2].wallet.signTransaction(transaction);
+          const error = predefined.INTERNAL_ERROR();
 
-        await Assertions.assertPredefinedRpcError(error, sendRawTransaction, false, relay, [signedTx, requestDetails]);
-      });
+          await Assertions.assertPredefinedRpcError(error, sendRawTransaction, false, relay, [
+            signedTx,
+            requestDetails,
+          ]);
+        });
+      }
```

Comment:
Note for reviewers: currently disable this test of fast return is turned on.

Open for Discussion: This test submits an `eth_sendRawTransaction` request with a calldata size of 60,000 bytes. Originally, this test would throw an SDKError indicating that the maximum file chunk size exceeds the limit of 20. However, with fast return enabled, no errors are thrown, only precheck errors. Currently, prechecks do not account for this.

The question is: what is a suitable solution to update this test? Should we revise the logic to state, "It should still return a transaction hash if the calldata exceeds the maximum file chunk size," since errors are no longer thrown when fast return is enabled? Or should we retain the existing logic and assert that the transaction receipt retrieved using the returned transaction hash will be null?

Reply:
Maybe we should check that the transaction receipt shows that the transaction was not successful due to chunk size
@quiet-node